apps/skilavottord/web/screens/RecyclingFund/Overview/Overview.tsx

```diff
@@ -59,19 +62,42 @@ const Overview: FC = () => {
               title: `${sidenavText.companies}`,
               link: `${routes.recyclingCompanies.baseRoute}`,
             },
+            {
```

Comment:
Missing check if we should show this or not ... should be haspermission accessControl